src/AccordionItem/index.tsx

```diff
@@ -17,6 +17,7 @@ export type AccordionItemProps = {
   id: string
   /** focuses the accordion item */
   focused?: number
```

Comment:
@CarlosRangel17 i guess my only ask would be to add in some descriptions for some for some of the Accordion Item props such as HidePreview, TitleInlineStyle and CenterInlineStyle

Reply:
Good catch Mo, I'll make an update accordingly.